CDCgov/prime-reportstream
# 9735 force using TLS 1.3 for NY-PHD

This PR forces TLS v1.2 for Rest Transport that uses a client authentication certificate.  We do not want receivers to be able to negotiate to an outdated TLS version which was possible in the previous implementation.  This is currently only used with NY-PHD but could be others in the future.

Test Steps:
1. Pull this branch to your local machine
2. Update your locally running ReportStream with NY-PHD production settings and credentials
3. Use the check functionality in the org UI to test connecting to the NY-PHD receiver
4. The dialog will let you know if everything works

## Changes
- Update SSL version in Rest Transport

## Checklist

### Testing

### Process

## Linked Issues
- Fixes https://github.com/CDCgov/prime-reportstream/issues/9735
